eth/beacon/db/chain.py

```diff
@@ -92,25 +96,49 @@ def persist_block_chain(
         raise NotImplementedError("BeaconChainDB classes must implement this method")
 
     #
-    # Raw Database API
+    # Crystallized State
     #
     @abstractmethod
-    def exists(self, key: bytes) -> bool:
+    def get_crystallized_state_by_root(self, state_root: Hash32) -> CrystallizedState:
```

Comment:
ntpick: I think we recently settled on a preference to just use `pass` here instead of  raising a `NotImplementedError` because the `ABC` module already takes care of not allowing these to be called, rendering the `raise` obsolete.